src/commands/transaction/construct_transaction/add_action_1/add_action/delete_account/mod.rs

```diff
@@ -37,3 +40,53 @@ impl From<DeleteAccountActionContext> for super::super::super::ConstructTransact
         item.0
     }
 }
+
+impl DeleteAccountAction {
+    pub fn input_beneficiary_id(
+        context: &super::super::super::ConstructTransactionContext,
+    ) -> color_eyre::eyre::Result<Option<crate::types::account_id::AccountId>> {
+        let beneficiary_account_id: crate::types::account_id::AccountId =
+            CustomType::new("What is the beneficiary account ID?").prompt()?;
+
+        if context.global_context.offline {
+            return Ok(Some(beneficiary_account_id));
+        }
+
+        #[derive(derive_more::Display)]
+        enum ConfirmOptions {
+            #[display(
+                fmt = "Yes, I want to check if account <{}> exists. (It is free of charge, and only requires Internet access)",
+                account_id
+            )]
+            Yes {
+                account_id: crate::types::account_id::AccountId,
+            },
+            #[display(fmt = "No, I know this account exists and want to continue.")]
+            No,
+        }
+        let select_choose_input =
+            Select::new("\nDo you want to check the existence of the specified account so that you don’t waste tokens with sending a transaction that won't succeed?",
```

Comment:
Apply the same suggestions for delete-account command that I requested above to all the add_action_* modules.

Reply:
Done